packages/webview_flutter/webview_flutter_android/lib/src/android_webview.dart

```diff
@@ -1150,6 +1170,25 @@ class WebChromeClient extends JavaObject {
     );
   }
 
+  Future<void> setHasJavaScriptAlertCallback(
```

Comment:
This and the two methods below should follow the same pattern as `onShowFileChooser` and be called `setSynchronousReturnValueForOnJsAlert`.